### PR TITLE
Converting the Change Increment ID KB to devdocs

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) converts a KB article into a devdoc. 

## Affected DevDocs pages

- http://devdocs.magento.com/guides/v2.2/config-guide/multi-site/change-increment-id.html

whatsnew
Added the [Change increment ID for a database entity on particular store](http://devdocs.magento.com/guides/v2.2/config-guide/multi-site/change-increment-id.html) topic to the Configuration Guide.

### CODE REVIEW

**Comment by dshevtsov** [comment]
@dobooth did you check the topics with `rake check`?

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+-  Some order records have been lost, but their IDs are already being used by payment gateways (like PayPal) for your current Merchant account. Such being the case, the payment  gateways stop processing new orders that have the same IDs, returning the "Duplicate invoice id" error
+
+{: .bs-callout-info}
+You may also fix the payment gateway issue for PayPal by allowing multiple payments per invoice ID in PayPal's Payment Receiving Preferences. See the Knowledge Base article: [PayPal gateway rejected request - duplicate invoice issue][] .
+
+## Prerequisite steps
+
+1. Find stores and entities for which the new increment ID should be changed.
+1. Connect to your MySQL DB.
+   For Magento Commerce (Cloud), at first, you need to SSH to your environment.
```
Comment (hguthrie) [inline guides/v2.2/config-guide/multi-site/change-increment-id.md]: Variable for Magento Commerce cloud is `{{site.data.var.ece}}`

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## When would you need to change increment ID
+
+You might need to change the increment ID for new DB entities in these cases:
+
+-  After a hard backup restore on a Live site
+-  Some order records have been lost, but their IDs are already being used by payment gateways (like PayPal) for your current Merchant account. Such being the case, the payment  gateways stop processing new orders that have the same IDs, returning the "Duplicate invoice id" error
+
+{: .bs-callout-info}
+You may also fix the payment gateway issue for PayPal by allowing multiple payments per invoice ID in PayPal's Payment Receiving Preferences. See the Knowledge Base article: [PayPal gateway rejected request - duplicate invoice issue][] .
```
Comment (hguthrie) [inline guides/v2.2/config-guide/multi-site/change-increment-id.md]:
```suggestion
You may also fix the payment gateway issue for PayPal by allowing multiple payments per invoice ID in PayPal's Payment Receiving Preferences. See [PayPal gateway rejected request - duplicate invoice issue][] in the _Knowledge Base_.
```

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+```sql
+ALTER TABLE sequence_order_1 AUTO_INCREMENT = 2000;
+```
+
+The next order placed at the store with `ID=1` will have the ID '#100002000'.
+
+## Additional recommended steps on production environments (cloud)
+
+Before executing the `ALTER TABLE` query on a production environment of Magento Commerce (Cloud), we strongly recommend performing these steps:
```
Comment (hguthrie) [inline guides/v2.2/config-guide/multi-site/change-increment-id.md]: Variable for cloud...

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+-  Test the entire procedure of changing the increment ID on your Staging environment
+-  [Create a DB backup][] to restore your Production DB in case of failure
+
+## Related documentation
```
Comment (hguthrie) [inline guides/v2.2/config-guide/multi-site/change-increment-id.md]: Boo...these related sections are draining and prone to URL maintenance. I would work links in to the content where it is relevant, or remove them completely.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+After executing the following query:
+
+```sql
+ALTER TABLE sequence_order_1 AUTO_INCREMENT = 2000;
+```
+
+The next order placed at the store with `ID=1` will have the ID '#100002000'.
+
+## Additional recommended steps on production environments (cloud)
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Don't use cloud in parenthesis.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+Before executing the `ALTER TABLE` query on a production environment of Magento Commerce (Cloud), we strongly recommend performing these steps:
+
+-  Test the entire procedure of changing the increment ID on your Staging environment
+-  [Create a DB backup][] to restore your Production DB in case of failure
+
+## Related documentation
```
Comment (hguthrie) [inline guides/v2.3/config-guide/multi-site/change-increment-id.md]: Boo

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  - Configuration
+  - System
+  - Setup
+---
+
+This article discusses how to change the increment ID for a Magento database (DB) entity (order, invoice, credit memo, etc.) on a particular Magento store using the `ALTER TABLE` SQL statement.
+
+## Affected versions
+
+-  Magento Commerce (on premise): 2.x.x
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Done

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+-  Magento Commerce (Cloud): 2.x.x
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Converted.

File: guides/v2.2/config-guide/multi-site/change-increment-id.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+-  MySQL: any supported version
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Good.

File: guides/v2.2/config-guide/multi-site/change-increment-id.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Changed!

File: guides/v2.2/config-guide/multi-site/change-increment-id.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Thanks

File: guides/v2.2/config-guide/multi-site/change-increment-id.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: Yup

File: guides/v2.2/config-guide/multi-site/change-increment-id.md (hunk 4 above)
Comment (dobooth) [reply guides/v2.2/config-guide/multi-site/change-increment-id.md]: I think they are ok.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+  - Configuration
+  - System
+  - Setup
+---
+
+This article discusses how to change the increment ID for a Magento database (DB) entity (order, invoice, credit memo, etc.) on a particular Magento store using the `ALTER TABLE` SQL statement.
+
+## Affected versions
+
+-  Magento Commerce (on premise): 2.x.x
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Thanks

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+-  Magento Commerce (Cloud): 2.x.x
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Done

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+-  MySQL: any supported version
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Done

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+-  Some order records have been lost, but their IDs are already being used by payment gateways (like PayPal) for your current Merchant account. Such being the case, the payment  gateways stop processing new orders that have the same IDs, returning the "Duplicate invoice id" error
+
+{: .bs-callout-info}
+You may also fix the payment gateway issue for PayPal by allowing multiple payments per invoice ID in PayPal's Payment Receiving Preferences. See the Knowledge Base article: [PayPal gateway rejected request - duplicate invoice issue][] .
+
+## Prerequisite steps
+
+1. Find stores and entities for which the new increment ID should be changed.
+1. Connect to your MySQL DB.
+   For Magento Commerce (Cloud), at first, you need to SSH to your environment.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Dun and Dun

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+1. Check the current auto_increment value for the entity sequence table using the following query:
+
+    ```sql
+    SHOW TABLE STATUS FROM `{database_name}` WHERE `name` LIKE 'sequence_{entity_type}_{store_id}';
+    ```
+
+## Example
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Removed them.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+## Update entity to change increment ID
+
+Update the entity using the following query:
+
+```sql
+ALTER TABLE sequence_{entity_type}_{store_id} AUTO_INCREMENT = {new_increment_value};
+```
+
+{: .bs-callout-info}
+Important: The new increment value must be greater than the current one, not less.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Nice!

File: guides/v2.3/config-guide/multi-site/change-increment-id.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Nobody puts baby in the corner.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md [hunk 16]
```diff
@@ -0,0 +1,84 @@
...
+
+```sql
+ALTER TABLE sequence_order_1 AUTO_INCREMENT = 2000;
+```
+
+The next order placed at the store with `ID=1` will have the ID '#100002000'.
+
+## Additional recommended steps on production environments (cloud)
+
+Before executing the `ALTER TABLE` query on a production environment of Magento Commerce (Cloud), we strongly recommend performing these steps:
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Yes

File: guides/v2.3/config-guide/multi-site/change-increment-id.md (hunk 6 above)
Comment (dobooth) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: Keeping it.

File: guides/v2.3/config-guide/multi-site/change-increment-id.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.3/config-guide/multi-site/change-increment-id.md]: 😜

**Comment by dshevtsov** [comment]
> General: The topics for 2.2 and 2.3 are identical. Is there a reason to not create a symlink?

**Comment by dobooth** [comment]
Yes, because I couldn't get the symlink to work correctly. So I made a separate topic, which will go away soon enough.

**Comment by dshevtsov** [comment]
@dobooth what was the issue with symlink?

**Comment by dobooth** [comment]
'User error' I'm sure.

**Comment by dobooth** [comment]
running tests

**Pull request merged**